XenModel/InvisibleMessages.resx

```diff
@@ -138,6 +138,9 @@
   <data name="HOMEPAGE" xml:space="preserve">
     <value>http://www.citrix.com/xenserver</value>
   </data>
+  <data name="HOMEPAGE_FILENAME" xml:space="preserve">
+    <value>HomePage.mht</value>

```

Comment:
And change the build scripts to bundle those files in the XenCenterResources.zip.